### ISSUE TITLE
Request for Final Results on CIFAR Dataset

### ISSUE THREAD

**Issue by leesin9527** [leftthomas/MoCo#6]
Thank you so much for sharing your code! 
I was wondering if you could share the final results you achieved on the CIFAR dataset? The results I obtained are shown in the image below.
![image](https://github.com/user-attachments/assets/340b2c71-4132-4af9-a5ed-7f7e79349399)

My training script is `python train.py --batch_size 1024 --epochs 50`, and other hyper parameters are default values.

**Comment by leftthomas** [comment]
@leesin9527 batch size too large, if you increase batch size, you should adjust learning rate
